src/OpenTelemetry.AutoInstrumentation/Configuration/TracerSamplerHelper.cs

```diff
@@ -21,7 +21,7 @@ namespace OpenTelemetry.AutoInstrumentation.Configuration;
 
 internal static class TracerSamplerHelper
 {
-    public static Sampler GetSampler(string tracerSampler, string tracerSamplerArguments)
+    public static Sampler? GetSampler(string? tracerSampler, string? tracerSamplerArguments)
```

Comment:
`tracerSampler` seems to be a candidate for refactor, no point to allow `null` value.

Reply:
It requires additional null check before, but fixed 16ce6223c259f140db177d5c08e2ca521f8366d2